### PR TITLE
Condition task 03/24

### PR DESCRIPTION
**Pull request by mkuleunich**
Hi Dima, please review.

### CODE REVIEW

**Comment by mkuleunich** [comment]
Hi Dima! Here is the 2nd task. Besides, there are some commits to solve conflicts I had during pushing changes to git - ignore them.
The commit for the review is 'Task 2 commit'.

Ping me via Skype in case of any questions.
Thanks!

P.S. I'm difficult case, but I'll survive)

File: tasks/cycles/cycles.js [hunk 1]
```diff
@@ -12,10 +16,65 @@
...
+{
+  const personIncomes = {
+    salary: 1985,
+    rent: -600,
+    interestOnDeposit: 250,
+    otherExpences: -300
+  };
+
+  for (let property in personIncomes) {
+    let sum = personIncomes[salary] + personIncomes[rent] + personIncomes[interestOnDeposit] + personIncomes[otherExpences];
```
Comment (dshaplyko) [inline tasks/cycles/cycles.js]: this also should've been don by cycle:

let sum = 0;
for (let elem in personIncomes) {
 sum += personIncomes[elem]
}

File: .jshintrc [hunk 2]
```diff
@@ -1,3 +1,8 @@
 {
+<<<<<<< HEAD
```
Comment (dshaplyko) [inline .jshintrc]: it is needed to resolve the conflicts by removing this kind of lines

File: tasks/tasks_condition_condition.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+ * Implement the logic when the system does appropriate operation based on passed operator param.
+ */
+
+
+    const x = 3;
+    const y = 7;
+    let operator = "add";
+
+    switch (operator) {
+          case operator = "add":
```
[the remark this answers is not in the document]
Comment (mkuleunich) [reply tasks/tasks_condition_condition.js]: fixed

File: tasks/cycles/cycles.js [hunk 4]
```diff
@@ -12,10 +16,65 @@
...
+   for (i = 1; i <= 10; i++) x = x * i;
+  console.log(x);
+}
+
+/**
+ * return concatenated string from an array of substring
+ */
+{
+  const substr = ["I", " love", " JS"];
+  console.log(substr.join (' '));
```
[the remark this answers is not in the document]
Comment (mkuleunich) [reply tasks/cycles/cycles.js]: Fixed

File: tasks/cycles/cycles.js (hunk 1 above)
Comment (mkuleunich) [reply tasks/cycles/cycles.js]: fixed

File: .jshintrc (hunk 2 above)
Comment (mkuleunich) [reply .jshintrc]: Fixed

**Comment by mkuleunich** [comment]
One more commit is added to this pull request - 'Fixed.'

Looks like I've done it! :)

**Pull request merged**